src/json-api/json-api-module-spec.ts

```diff
@@ -203,17 +209,21 @@ describe('json-api-module', () => {
         email: 'foo@goomail.com',
         commentId: 4,
       };
-      const commentActivity = Object.assign({}, minardActivity, attributes);
+      const commentActivity = {
+        ...minardActivity,
+        ...attributes,
+      };
```

Comment:
If you define these in the following way there's no need or the `as any` cast below:

```js
      const attributes: Partial<MinardActivity> = {
        activityType: 'comment',
        name: 'foo',
        message: 'foo msg',
        email: 'foo@goomail.com',
        commentId: 4,
      };
      const commentActivity: MinardActivity = {
        ...minardActivity,
        ...attributes,
      };
```